src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocument.cs

```diff
@@ -43,6 +43,22 @@ internal class ManifestDocument
 			{ "action",   "android.intent.action.MAIN" },
 			{ "category", "android.intent.category.LAUNCHER" },
 		};
+
+		// the elements which we apply the "." -> PackageName replacement on
+		static readonly string [] ManifestElementFixupNames = {
```

Comment:
I thought about this method. It would be more explicit (yay) but would probably result in us having to repeatedly update with new items as they are found/introduced (boo). 

So I'm torn.